mne/forward/tests/test_make_forward.py

```diff
@@ -237,6 +238,9 @@ def test_make_forward_solution():
     with pytest.raises(RuntimeError, match='homogeneous.*1-layer.*EEG'):
         make_forward_solution(fname_raw, fname_trans, fname_src,
                               fname_bem_meg)
+    # make sure everything can be path-like (gh #10872)
+    _ = make_forward_solution(Path(fname_raw), Path(fname_trans),
+                              Path(fname_src), Path(fname_bem), mindist=5.)
```

Comment:
Forward creation is generally slow, but we do it a lot of places. It's probably better time-wise to just `Path(...)`-wrap above rather than recompute in a separate call